### DISCUSSION TITLE
Clockwork Beta 14 - Release Announcement

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#51] [Announcements]
Hi there! Clockwork 2.0.0.0-beta14 has now been released, and with it a ton of new features! Some big highlights include:

- Clockwork's UI has been fully revamped, with a sleek modern style. 
![clockwork-1](https://github.com/red-stone-network/clockwork/assets/91223726/692f2335-a25d-4e75-9562-7ed3d45b37df)
- Wallpapers have been added! By default, Clockwork comes with 4 wallpapers, but you can add your own if you upload it to another website!
- There's now a prompt when closing the tab, which gets around certain blocking systems that just redirect you to a page.
- All the icons have been remade, including the Clockwork logo.

This update should be out to all instances now! You can find the update logs [here.](https://github.com/red-stone-network/clockwork/blob/main/UPDATELOGS.md)

**Comment by Ender-Ghost-Craft** [comment]
can you guys make a guide to make a theme on the wiki?

**Reply by adonzone** [reply]
wheres the wiki?

**Reply by Ender-Ghost-Craft** [reply]
the 6th tab

**Comment by Ender-Ghost-Craft** [comment]
can you also make it so that when you hover over an icon, it plays an animated gif

**Comment by keeblerkiller** [comment]
how do I start it?
